tests/testthat/test-custom-remote.R

```diff
@@ -0,0 +1,12 @@
+context("remote")
+
+setwd("./travis-testthat")
```

Comment:
This looks dangerous. Can we use `withr::local_dir()` inside the test instead?

Reply:
Replaced all `setwd()´ by 

```r
withr::with_dir(
    here::here("tests/testthat/travis-testthat"),
    {

```

Reply:
I still like `local_dir()` much better, because this reduces indent.